<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JackParameterForm</class>
 <widget class="QWidget" name="JackParameterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="driverLabel">
     <property name="text">
      <string>Driver</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="driverBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>In Device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qjackctlInterfaceComboBox" name="iFaceBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>(default)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Out Device</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="qjackctlInterfaceComboBox" name="oFaceBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>(default)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_srChoice_2">
     <property name="text">
      <string>Sample Rate:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="m_srChoice">
     <item>
      <property name="text">
       <string>22050</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>44100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>48000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>88200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>96000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>192000</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="m_fppLabel">
     <property name="text">
      <string>Frames/Period:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="m_fppChoice">
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2048</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4096</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qjackctlInterfaceComboBox</class>
   <extends>QComboBox</extends>
   <header>qjackctlinterfacecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
